### DISCUSSION TITLE
Whether the teleport community version has access rights to specific servers for each user or not (RBAC) ?

### DISCUSSION THREAD

**Discussion by longansv** [gravitational/teleport#39225] [Q&A] [answered]
On my teleport community edition, there are many users and many node servers. Can I configure it so that a specific user only has permission to view and access specific server nodes?

**Comment by stevenGravy** [comment, marked as answer]
Yes you can set the labeling and logins allowed. Servers will require labeling in order to allow or deny specific nodes. https://goteleport.com/docs/access-controls/getting-started/#step-33-create-a-custom-role

**Reply by longansv** [reply]
yes, thank you very much
